src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/lib/can_append_wildcard.js

```diff
@@ -0,0 +1,7 @@
+export const canAppendWildcard = ({ data: keyPressed }) => {
```

Comment:
Why destructure an object here if there's only one prop? Why not pass in the `keyPressed` argument directly?

Reply:
You're right, no need